kv/local_test_cluster.go

```diff
@@ -146,5 +122,8 @@ func (ltc *LocalTestCluster) Start(t util.Tester) {
 
 // Stop stops the cluster.
 func (ltc *LocalTestCluster) Stop() {
+	if r := recover(); r != nil {
```

Comment:
FYI this doesn't work very well. `t.Fatal()` calls `runtime.Goexit`, which is equivalent to `panic(nil)`, which you cannot detect with `recover()`. So it only partially solves the "hang at shutdown" problem. If we had the `testing.T` available we could do `if t.Failed() { return }`.